go.mod

```diff
@@ -1,35 +1,38 @@
 module redhat-developer/red-hat-developer-hub-operator
 
-go 1.21
+go 1.22.6
```

Comment:
We can not bump it to 1.22 due to
https://catalog.redhat.com/software/containers/rhel9/go-toolset/61df08166d9a1b7b2aab2344